### ISSUE TITLE
pre-listening on Mixxx

### ISSUE THREAD

**Issue by djleosapao** [mixxxdj/mixxx#13053]
### Bug Description

Hello everyone, from Mixxx Dj support

I'm having some issues with pre-listening on Mixxx, it's basically not working for me! I'm using the Numark Mixtrack pro Fx controller, and everything else seems to be working perfectly, so I assume the settings are wrong somewhere!

I tried some different combinations in options>preferences>sound hardware>headphones, but no signal sent to the headphones, for it to work I have to restart the operating system and do all the configuration again or unplug the controller cable and plug it in again , there are times when I can only hear the music playing through the speakers, but now I don't receive anything in return on the Tecnichs 1210 headphones that I use.

I thank you for your attention.

### Version

2.40

### OS

Linux ( Zorin OS 17.1

**Comment by ronso0** [comment]
The headphone signal is sent to the Headphones output.
It combines signals from
* all decks / mics / aux that have Pfl engaged
* preview deck

On controllers with built-in (4-channel!) soundcards the headphone output is usually channels 3/4.
So, for the Pro FX you need to set Main out to **Numark Mixtrack Pro FX** Channels 1/2 and
Headphones to **Numark Mixtrack Pro FX** Channels 3/4.

**Comment by ronso0** [comment]
Actually, I'm pretty sure this is not a Mixxx bug but just a question of configuration, i.e. a support request.
For those there is the Mixxx forum -> https://mixxx.discourse.group/c/support/6

FYI Github Issues are only for bugs or feature request (don't post them in the forum, they'll likely get lost / overlooked).

**Comment by djleosapao** [comment]
use a soundboard to do live streaming on YouTube, I normally add the soundboard to channels 1 and 2, but when I select the bottom part where the headphones are, I try to add the controller to channels 3 and 4 but it gives me an error.

**Comment by ronso0** [comment]
Let's first try to get sound out of the Pro FX' headphone jack, right?
Does that work with channels 3/4?

It would be helpful if you'd let us know which error you get.

**Comment by djleosapao** [comment]
Yes, when I install it on channels 3 and 4 I normally set it to default and it works normally, but when I change the sound settings again to update any changes it stops sending a signal to the headphones so I can't listen to the music on either side. right and left side.
[Abrir no Google](https://translate.google.com/?gs_lcrp=EgZjaHJvbWUqCQgCECMYJxjqAjIJCAAQIxgnGOoCMgkIARAjGCcY6gIyCQgCECMYJxjqAjIJCAMQIxgnGOoCMg8IBBAuGCcYxwEY6gIY0QMyCQgFECMYJxjqAjIJCAYQIxgnGOoCMgkIBxAjGCcY6gLSAQkyNjA5ajBqMTWoAgiwAgE&um=1&ie=UTF-8&hl=pt&client=tw-ob#pt/en/Sim,quando%20instalo%20nos%20canais%203%20e%204%20normalmente%20ponho%20em%20default%20ai%20funciona%20normal,%20mas%20quando%20mexo%20de%20novo%20nas%20configura%C3%A7%C3%B5es%20de%20som%20para%20atualizar%20alguma%20altera%C3%A7%C3%A3o%20ele%20para%20de%20mandar%20sinal%20para%20o%20headphones%20assim%20n%C3%A3o%20consigo%20ter%20P%C5%95e-escuta%20das%20musicas%20tanto%20do%20lado%20direito%20e%20do%20lado%20esquerdo.)

**Comment by djleosapao** [comment]
Yes, when I install it on channels 3 and 4 I normally set it to default and it works normally, but when I change the sound settings again to update any changes it stops sending a signal to the headphones so I can't listen to the music on either side. right and left side.

**Comment by ronso0** [comment]
> I normally set it to default

What does that mean?

> but when I change the sound settings again to update any changes it stops sending a signal to the headphones

What changes? Is the "soundboard" now involved?
Please let us know what kind of device that is and how you intend to route signals from Mixxx.
Screenshot of the sound preferences would also be helpful.

**Comment by djleosapao** [comment]
![Captura de tela de 2024-04-05 13-50-49](https://github.com/mixxxdj/mixxx/assets/162135312/8561e761-4384-46bb-b353-f9255472485e)
![WhatsApp Image 2024-04-05 at 13 46 43](https://github.com/mixxxdj/mixxx/assets/162135312/93baf2f2-068d-4d18-8ce5-b61d3502e0c2)

I normally use this soundboard to do lives, in the configuration I normally use, I include the soundboard on channels 1 and 2, it works smoothly. Now in the lower part where the phone is located, I select the default one that works, now in the case when I try to select the controller it does not accept.

**Comment by ronso0** [comment]
What exactly do you want to mix with the Xenyx?

IIUC it provides 2 **input** channels (one stereo pair) via USB and some wired inputs/outputs.
(Please verify! If you select the Xenyx device in Mixxx how many channels does it list?)

If that is true, you can only send one output to the Xenyx USB input. Use the Mixxx Main out since you probably want use the LINE fader on the Xenyx.
For the headphones you may use the Pro FX' headphone output and connect that to **2-TRACK** jacks on the Xenyx (1/2 jack -> RCA), set the  **2-TRACK TO** switch to **PHONES**, et voila:
* Mixxx main on the LINE
* Mixxx headphones on the PHONES

**Comment by djleosapao** [comment]
Thank you, I will follow these explanations but still.
